test/recipes/controls/aws_parallelcluster_slurm/install_slurm_spec.rb

```diff
@@ -132,3 +132,33 @@
     end
   end
 end
+
+control 'pam_slurm_adopt_module_installed' do
+  title "Check that pam_slurm_adopt has been built and installed"
+
+  lib_security_folder = '/lib/security/'
+  if os.redhat?
```

Comment:
Does this mean only for RHEL or for the RedHat family?

Reply:
redhat family.. I actually tested on Amazon Linux